packages/web-components/src/components/va-alert/va-alert.tsx

```diff
@@ -127,8 +133,16 @@ export class VaAlert {
   }
 
   render() {
-    const { backgroundOnly, status, visible, closeable } = this;
-    const classes = `alert ${status} ${backgroundOnly ? 'bg-only' : ''}`;
+    const {
+      backgroundOnly,
+      status,
+      visible,
+      closeable,
+      showIconInBackgroundOnly,
+    } = this;
+    const classes = `alert ${status} ${backgroundOnly ? 'bg-only' : ''} ${
+      backgroundOnly ? (showIconInBackgroundOnly ? '' : 'hide-icon') : ''
+    }`;
```

Comment:
This is getting a bit harder to read with the addition of more ternaries and template strings. Could we use the `classnames` dependency for this, maybe like:

```js
const classes = classnames(
    'alert',
    status, 
    {
      'bg-only': backgroundOnly,
      'hide-icon': backgroundOnly && showIconInBackgroundOnly
    })
```